powerauth-webflow/src/main/java/io/getlime/security/powerauth/app/webflow/configuration/WebMvcConfiguration.java

```diff
@@ -132,6 +135,16 @@ public ReloadableResourceBundleMessageSourceWithListing messageSource() {
         return messageSource;
     }
 
+    @Bean
+    public Resource languageSettingSource() {
+        try {
+            return new UrlResource(configuration.getResourcesLocation() + "lang.json");
+        } catch (MalformedURLException ex) {
```

Comment:
You may throw the exception, it stops starting of Spring context a will be reported out of the box.